### ISSUE TITLE
DWARF objects (*.dwo) are not cleaned

### ISSUE THREAD

**Issue by clopez** [icecc/icecream#140]
The use case is the following:

* I'm building with Clang, and enabling [DebugFission](https://gcc.gnu.org/wiki/DebugFission).
* I'm also sending my toolchain. Created  like this:
```
 export ICECC_VERSION="$(pwd)/$(/usr/lib/icecc/icecc-create-env --clang /usr/bin/clang /usr/lib/icecc/compilerwrapper --addfile /usr/bin/objcopy|tee /dev/stderr|grep creating|awk '{print $2}')"
```
* [I'm building WebKitGTK+ in Debug mode.](https://trac.webkit.org/wiki/WebKitGTK/SpeedUpBuild#Usingiceccwithclang)

Everything works as expected. 

The problem is that on the build slaves, a lot of trash is left there. On the folder ```/var/cache/icecc/target\=x86_64/clang-toolchain/tmp/``` there are thousands of files with names like
```
[...]
icecc-10280_187bfb32.dwo
icecc-10281_5af2fb32.dwo
icecc-10282_620afb32.dwo
icecc-10283_eba8fb33.dwo
icecc-10284_a51bfb33.dwo
[...]
```

```
# du -hs /var/cache/icecc/
28G	/var/cache/icecc/
# find /var/cache/icecc/ -type f -name "*.dwo" |wc -l
14178
# find /var/cache/icecc/ -type f -name "*.dwo" -delete
# du -hs /var/cache/icecc/
310M	/var/cache/icecc/
```

And this has been created in less than one week.

This causes that the buildslaves run out of disk space very quick.

I have workarounded it by deploying this cronjob

*  ```/etc/cron.hourly/delete-icecc-trash``` 
```
#!/bin/bash
# Deletes debug-fission objects older than 5 minutes. Runs each hour.
find /var/cache/icecc/ -type f -mmin +5 -name "*.dwo" -delete
```

But I guess icecc should be responsible of deleting the .dwo objects when they are not longer needed. What do you think?

**Comment by clopez** [comment]
An update:

* Seems icecc is not copying back the dwo objects from the buildslaves.
* Only the dwo objects that were built locally are preserved.

**Comment by johnmiked15** [comment]
This would be resolved by #144.

**Comment by johnmiked15** [comment]
Resolved by #144
